.gitignore

```diff
@@ -1 +1,2 @@
 tmp
+.idea
```

Comment:
I'm ok with this addition, but could you please move it into its own commit so that changes to the file dont get lost in the minutia of other issues and work ?